code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -691,9 +709,396 @@
 		string_of_knowledge += knowledge.name
 
 	parts += english_list(string_of_knowledge)
+	parts += get_flavor(cultiewin, ascended, heretic_path) //BUBBERSTATION EDIT- SOURCED FROM YOG
 
 	return parts.Join("<br>")
 
+/datum/antagonist/heretic/proc/get_flavor(cultiewin, ascended, heretic_path) //HUGE BUBBERSTATION EDIT, TAKEN FROM YOGS, START HERE
+	var/list/flavor = list()
+	var/flavor_message
+
+	var/alive = owner?.current?.stat != DEAD
+	var/escaped = ((owner.current.onCentCom() || owner.current.onSyndieBase()) && alive)
+
+	flavor += "<div><font color='#6d6dff'>Epilogue: </font>"
+	var/message_color = "#ef2f3c"
+
+	//Stolen from chubby's bloodsucker code, but without support for lists
+
+	if(heretic_path == PATH_ASH) //Ash epilogues
```

Comment:
cant this be a switch statement

Reply:
If one would be so inclined to, yes. It works and I'd have to either make a proc for getting the path or rewrite it I guess

Reply:
maybe tie the flavour text to each heretic's ascensions knowledge

Reply:
> maybe tie the flavour text to each heretic's ascensions knowledge

Ascencion knowledge? But not every heretic will buy the final knowledge since it takes fulfilling obj to get it.